Websocket proxy nginx conf fix.

qraxiss opened the pull request:
https://github.com/flet-dev/flet/pull/748#event-8119098225

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/flet-dev/website?pullRequest=116) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/flet-dev/website?pullRequest=116) before we can accept your contribution.<br/><hr/>**Mehmet Fatih Güman** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/flet-dev/website?pullRequest=116) it.</sub>

FeodorFitsner commented:
What issue does this PR fix (looks like you posted a link to unrelated discussion)?

qraxiss commented:
> What issue does this PR fix (looks like you posted a link to unrelated discussion)?

My websocket problem in nginx was solved when I replaced keep-alive with "upgrade". Other changes were accidental, I guess. If you think that the "upgrade" solution is correct, I can throw another PR and eliminate what I changed wrong.

FeodorFitsner commented:
I've checked the number of sources and looks like `proxy_set_header Connection "upgrade"` is the correct solution which worked for you.
